docs/devenv_rhel8.md

```diff
@@ -60,7 +60,7 @@ Run the following commands to configure SUDO, upgrade the system, install basic
 echo -e 'microshift\tALL=(ALL)\tNOPASSWD: ALL' | sudo tee /etc/sudoers.d/microshift
 sudo dnf clean all -y
 sudo dnf update -y
-sudo dnf install -y git cockpit make gcc selinux-policy-devel rpm-build bash-completion
+sudo dnf install -y git cockpit make gcc selinux-policy-devel rpm-build bash-completion go-toolset
```

Comment:
Why is this package needed? Also, if we change anything here, automation script need to be updated.

Reply:
It installs golang, otherwise `make` won't build the microshift executable, it throws error about `go` not present on the system (atleast in RHEL 8.7)

Reply:
Right. We missed reverting this line when we moved to go installation from RPM. The command used in the automation is the correct one (see https://github.com/openshift/microshift/blob/main/scripts/devenv-builder/configure-vm.sh#L47) 

```suggestion
sudo dnf install -y git cockpit make golang selinux-policy-devel rpm-build bash-completion
```

Reply:
Done.